### DISCUSSION TITLE
MAC

### DISCUSSION THREAD

**Discussion by retepterhorst** [Open-Systems-Pharmacology/Forum#1209] [Feature Requests & Ideas]
why isn't the suite available for MAC???

**Comment by msevestre** [comment]
The suite is implemented using .net technology. While the core can be migrated to another environment (which we did already as it is working on Linux), the user interface is only available on windows.
If you absolutely need to use a mac, you can easily used the suite by using a solution such as parallel, allowing you to install a Windows environment on a mac os. Once this is done, simply install the suite on your Windows instance.

**Reply by zt1947** [reply]
Thanks!
Has anyone implemented parallel or Crossover to run OSP on Mac?

**Comment by StephanSchaller** [comment]
@retepterhorst , we always take on sponsors to implement new features. If you want MAC, find someone with a budget and we will be happy to make it a reality.

**Comment by Yuri05** [comment]
Another option you might try: https://www.codeweavers.com/crossover 
No windows license required. You could use the free trial to check if OSP Suite runs in this environment.

And if the OSP Suite is not running yet - you could submit a request to the developers: https://www.codeweavers.com/compatibility/submit

**Reply by msevestre** [reply]
Interesting... I'll try on my Mac to see if this works

**Reply by zt1947** [reply]
Did it work?
